portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -89,6 +90,28 @@ window.historyType = 'memory';
 
 window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
 
+window.priorityStudies = {};
+<%
+List<String[]> priorityStudies = GlobalProperties.getPriorityStudies();
```

Comment:
I tried it but it turns out to be a hassle with the tag file.  It was more work than it was worth.  I think we should just continue to work toward applying the template to all pages, which will solve the DRY issue once and for all.